system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -217,6 +248,83 @@
                 :VariableLevel1 "EMISSIONS"}]
               (:ScienceKeywords (:umm concept)))))))
 
+(deftest bulk-update-remove-all-platforms-test
+  (let [concept-ids (ingest-collection-in-umm-json-format find-remove-all-platforms-instruments-umm)
+        _ (index/wait-until-indexed)
```

Comment:
In light of the above comment, the assignment `concept-ids` can be in its own let, then you can just do a regular `(index/wait...)`, then a check or two for the presence of the data that you will be removing.

Reply:
I misread this -- sorry! You're right -- this is actual ingest with no removal.